docs/hints-for-use-with-docker.md

```diff
@@ -57,3 +57,19 @@ To authenticate with a private Git repository, ORT uses the (semi)standardized `
    ```
 
    **Important:** Ensure that the `.netrc` file has been created at `netrc_folder_path` before running the command, otherwise Docker will create a folder `.netrc` inside the container, instead of mounting a single file.
+
+It is also possible to use `git config` to setup access to private repositories.
+
+1. Configure your gitconfig with your github credentials
+
+    ```shell
+    [url "https://<username>:<password>@github.com"]
+	         insteadOf = /https://github.com    
```

Comment:
I think the leading `/` in the URL is incorrect.

Reply:
Removed it.